SPECS/ruby/ruby.spec

```diff
@@ -13,6 +13,12 @@ BuildRequires:  openssl-devel
 BuildRequires:  readline
 BuildRequires:  readline-devel
 BuildRequires:  tzdata
+
```

Comment:
Looks like the spec-linter would like the Requires for gmp and openssl above the "if with check". Not sure if this is critical. @thcrain-msft thoughts?

Reply:
This doesn't seem to be critical, and Pawel's change is actually more in line with some changes I'm working on for the linter.

Reply:
@rlmenge: I did these changes on purpose. Like Thomas said, this is closer to what we're going for anyway. The linters is being actively updated to best fit our preferences and needs, so for now I'm not yet in the "blindly trust the linter" mode, but we'd like to get to that point (or close to it) soon.:) For now I'm taking a look at what the linter suggests and pick the things I agree with.